### ISSUE TITLE
Fix grammatical and formatting issues

### ISSUE THREAD

**Issue by samrobbins85** [samrobbins85/notes-site#54]
This is an umbrella issue that if you can find any grammar or formatting issues, a PR to fix them would be much appreciated

**Comment by jeannienguyen** [comment]
Hey, @samrobbins85 - I just created a PR for this issue. Thank you!

**Comment by samrobbins85** [comment]
Thanks, a great PR, have a good Hacktoberfest!

**Comment by siddharthdeo99** [comment]
Is this issue still open to contribute?? If yes i will give it a look.

**Comment by samrobbins85** [comment]
Yeah I'm going to leave it open for hacktoberfest as there's no way I can tell when there's no longer any issues. Thanks very much
